content/introduction-and-setup/walkthrough/cloning-a-project/index.md

```diff
@@ -0,0 +1,104 @@
+---
+title: "Cloning a C# Project"
+date: 2022-12-15T09:16:07-06:00
+draft: false
+weight: 5
+originalAuthor: Sally Steuterman # to be set by page creator
+originalAuthorGitHub: gildedgardenia # to be set by page creator
+reviewer: # to be set by the page reviewer
+reviewerGitHub: # to be set by the page reviewer
+lastEditor: # update any time edits are made after review
+lastEditorGitHub: # update any time edits are made after review
+lastMod: # UPDATE ANY TIME CHANGES ARE MADE
+---
+
+Follow along with the provided walkthroughs to learn how to clone repos into Visual Studio. Try it out on your own by cloning the [LaunchCodeEducation/csharp-web-dev-datatypes](https://github.com/LaunchCodeEducation/csharp-web-dev-datatypes) repository page and fork the repository into your own GitHub account by selecting Fork from the top right of the page.
+
```

Comment:
Wouldn't we want to fork the [LaunchCodeEducation/csharp-web-dev-datatypes](https://github.com/LaunchCodeEducation/csharp-web-dev-datatypes) repository to our own account and then clone that repo following the instructions provided?

Reply:
Done!